packages/components/organisms/f-header/src/components/CountrySelector.vue

```diff
@@ -110,4 +110,6 @@ export default {
 };
 </script>
 
-<style lang="scss" module src='../assets/scss/navigation.scss' />
+<style lang="scss" module>
+@import '../assets/scss/navigation.scss';
```

Comment:
I think @tommcclean was saying these style imports need to go in the script section

Reply:
Just tried this and it broke. I suspect it only works when you're importing external CSS rather than internal SCSS, because some of the styles from this SCSS module are referenced inside these components so it needs to be visible to the SCSS processor.